tests/linalg_test.py

```diff
@@ -1679,5 +1679,42 @@ def testSchurBatching(self, shape, dtype):
       Ts, Ss = vmap(lax.linalg.schur)(args)
       self.assertAllClose(reconstruct(Ss, Ts), args, atol=1e-4)
 
+  @jtu.sample_product(
+      cshape=[(4,), (8,), (3, 7)],
+      cdtype=float_types + complex_types,
+      rshape=[(3,), (7,), (2, 1, 4)],
```

Comment:
You might also test shapes `()` and `(0,)` to make sure you're covering all corner cases.

Reply:
done